README.md

```diff
@@ -16,6 +16,12 @@ the easiest way to add a package and its dependencies is to use:
 python3 -m add_pkg PKGNAME
 ```
 
+You will need to install the right dependencies first though:
+
+```bash
+pip install -r docker/requirements.txt
+```
+
```

Comment:
the repository uses tox, so the conventional way is to use tox's devenv feature (which I don't really want to duplicate docs for how to use tox here)